### PR TITLE
Add default sort option and options blacklist

### PR DESCRIPTION
**Pull request by alexbridge**
Add default sort option and options blacklist

### CODE REVIEW

File: extension-config.json [hunk 1]
```diff
@@ -10,6 +10,24 @@
         "type": "json" ,
         "label": "Map for sort options"
       }
+    },
+    "defaultSortOption": {
```
Comment (Carsten89) [inline extension-config.json:14]: fyi in CURB we have already a setting for the default sort order.

File: frontend/products/subscriptions.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+import { SEARCH_PATTERN } from '@shopgate/engage/search';
+import config from '../config';
+
+const { defaultSortOption } = config;
+
+/**
+ * Subscriptions
+ * @param {Function} subscribe subscribe
+ */
+export default (subscribe) => {
```
Comment (Carsten89) [inline frontend/products/subscriptions.js:12]: maybe it would be better to add config for this to themes

File: extension-config.json (hunk 1 above)
Comment (alexbridge) [reply extension-config.json:14]: Sounds nice

File: frontend/products/subscriptions.js (hunk 2 above)
Comment (alexbridge) [reply frontend/products/subscriptions.js:12]: Since its custom sort options, themes do not know what it will be

Comment (Carsten89) [reply frontend/products/subscriptions.js:12]: valid point. So we should wait until we migrated this ext to CURB. Then we can use the settings there as default here + you can overide it with setting from here

**Review by Carsten89 — APPROVED** [review]

**Pull request merged**